Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/program/model/address/AddressMapImpl.java

```diff
@@ -222,7 +222,7 @@ public int findKeyRange(List<KeyRange> keyRangeList, Address addr) {
 	}
 
 	/**
-	 * @see ghidra.program.model.address.AddressMap#getKeyRanges(ghidra.program.model.address.AddressSetView)
+	 * @see ghidra.program.database.map.AddressMap#getKeyRanges(AddressSetView, boolean)
 	 */
```

Comment:
Shouldn't this class be implementing AddressMap? It seems strange that it isn't. If it's supposed to be these can just be removed.

Reply:
Maybe?  It currently doesn't implement all of the methods though.  What you did for this PR is fine.